code/controllers/subsystem/air.dm

```diff
@@ -222,8 +222,8 @@ SUBSYSTEM_DEF(air)
 		src.currentrun = networks.Copy()
 	//cache for sanic speed (lists are references anyways)
 	var/list/currentrun = src.currentrun
-	while(currentrun.len)
-		var/datum/thing = currentrun[currentrun.len]
+	while(length(currentrun))
```

Comment:
if currentrun goes null this change would hide it where as the old version would runtime.

Things to think about.

Reply:
it will still runtime on line 227

Reply:
227 wouldn't run because `length(currentrun)` on null returns 0 and this will fail the while check

Only a byond bug would lead to currentrun being null anyways, so this a moot point, but if the copy() fails on a valid list for some reason, the observed behavior is that it would just fail the while check and then reach the end of the proc.

Reply:
ah yeah youre right